packages/client/src/generation/generateClient.ts

```diff
@@ -317,7 +317,9 @@ export async function generateClient(options: GenerateClientOptions): Promise<vo
   if (getClientEngineType(generator) === ClientEngineType.Wasm) {
     const queryEngineWasmFilePath = path.join(runtimeSourceDir, 'query-engine.wasm')
     const queryEngineWasmTargetPath = path.join(finalOutputDir, 'query-engine.wasm')
-    await fs.copyFile(queryEngineWasmFilePath, queryEngineWasmTargetPath)
+
+    const copyOrSymlink = testMode ? fs.symlink : fs.copyFile
+    await copyOrSymlink(queryEngineWasmFilePath, queryEngineWasmTargetPath)
```

Comment:
Avoids spending time on making copies.

Reply:
I'm not sure if that works well on Windows 🤔

Reply:
I think you might be right. I recall issues with rights to create symlinks on Windows. Once we hook it on CI if there are any issues, I'll enable `copyFile` for windows instead.